specification/resources/apps/examples/curl/apps_update.yml

```
@@ -1,6 +1,5 @@
-lang: cURL
-source: |-
-  curl -X PUT \
-    -H "Content-Type: application/json" \
-    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
-    "https://api.digitalocean.com/v2/apps/{id}"
+curl -X PUT \
```

```suggestion
lang: cURL
source: |-
  curl -X PUT \
```

This is a yaml file. The example needs to be under the `source` key.

---

ah, got it. Thanks!